doc/DeveloperGuide.md

```diff
@@ -88,6 +88,44 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**Preconditions**
+
+1. AddressBook confirms the change with the user before carrying out the operation
+
+**MSS**
+
+1. AddressBook requests user for tag to rename
+2. User enters tag to rename
+3. AddressBook shows the list of persons with tag
+4. AddressBook requests user for new tag
```

Comment:
use cases capture the back and forth interaction between the system and the user, so there should be something that the user does in between 3 & 4